runtime/commands/taglogic.js

```diff
@@ -116,6 +116,25 @@ Commands.tag = {
             msg.channel.sendMessage('`' + cp + '`')
           }
         })
+      } else if (index[0].toLowerCase() === 'list') {
+        var author = msg.author
+        if (index[1] && msg.mentions.length === 1) {
+          author = msg.mentions[0]
+        }
+        r.db('Discord').table('Tags').filter({owner: author.id}).count().run().then((c) => {
+          if (c === 0) {
+            msg.channel.sendMessage(`${msg.author.id === author.id ? "You don't" : 'This user does not'} have any tags!`)
+            return
+          } else {
+            var tagsArray = []
+            r.db('Discord').table('Tags').filter({owner: author.id}).run().then((tags) => {
+              for (var i in tags) {
+                tagsArray.push(tags[i].id)
+              }
+              msg.channel.sendMessage(`Found ${c} tags for **${author.username}**:\n${tagsArray.join(', ')}`)
```

Comment:
Don't try to send the message if the result would be over 2000 characters long, try to error out instead.